core/src/apps/bitcoin/sign_taproot.py

```
@@ -124,7 +127,10 @@ async def sign_taproot(
                 coin.b58_hash,
             )
         elif out.is_opreturn():
-            assert out.nValue == 0, "OpReturn output should have 0 value"
+            if out.nValue != 0:
+                assert (
```

Ruff (0.11.9)

Use of `assert` detected

(S101)